messages/run-summary-viewer.md

```diff
@@ -0,0 +1,27 @@
+# summary.header
+
+Summary
+
+# summary.found-no-violations
```

Comment:
Remind me the process - When are these reviewed by @jshackell-sfdc ?

Reply:
Good catch. That hasn't happened yet. These were all suggested by Marcelino, but I should get Juliet to take a look at them too.